Util/openl-maven-plugin/it/openl-openapi-verify-positive/pom.xml

```diff
@@ -0,0 +1,31 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.openl.internal</groupId>
+    <artifactId>openl-openapi-verify-positive</artifactId>
+
+    <version>0.0.0</version>
+    <packaging>openl</packaging>
+
+    <name>OpenL Plugin: OpenL OpenAPI Verify Positive Test</name>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <org.openl.version>@project.version@</org.openl.version>
+    </properties>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.openl.rules</groupId>
+                <artifactId>openl-maven-plugin</artifactId>
+                <version>${org.openl.version}</version>
+                <extensions>true</extensions>
+                <configuration>
+                    <sourceDirectory>./</sourceDirectory>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
+</project>
```

Comment:
Is it possible to check that OpenAPI validation was processed successfully? In verify.groovy

Reply:
We can just check that the build was successful. We only see the OpenAPI error log.